### DISCUSSION TITLE
ein-get as organized clone?

### DISCUSSION THREAD

**Discussion by aohoyd** [GitoxideLabs/gitoxide#698] [Ideas]
Found this impressive project yesterday and this came to mind. `ein` has a tool `organize` and would possibly get `clone`. What do you think about `ein clone --origanized` or `ein get` (like [git-get](https://github.com/grdl/git-get))? 

And where is the development of a `ein clone` happening?

**Comment by Byron** [comment]
Indeed, something that could clone and organize would be an interesting addition to `ein`, and something that can be done today and with some confidence (as in: it seems to be the right thing and an improvement over the status quo).

> And where is the development of a `ein clone` happening?

I don't think I understand the question, but trying the obvious answer: [it's here](https://github.com/Byron/gitoxide/blob/4c93de113898c6ff6ae1da3ea38d646626c86205/src/porcelain/main.rs#L26). Also note that clone is already implemented (even though checkouts are not completely done yet) in `gix clone` - adding an `--organized` flag there would probably be easiest as it would only have 'devleopment-tool' level of quality there. With `ein` I have to think about the greater workflow that would be affected by even seemingly straightforward sub-commands like `get` to figure out if it truly is a good addition.

**Reply by aohoyd** [reply]
I just thought `ein clone` was being developed somewhere in another branch. Anyway, you have answered my question, thanks! I'll add the option to `gix clone`, but I need some time to figure things out :)
